test/test_config.py

```diff
@@ -22,10 +29,26 @@ def get_test_config():
                 "reconstruction_dir": "my_dir"
             }
         },
-        "datasets": [{"label": "alg1"}, {"label": "alg2"}],
-        "gold_standards": [{"label": "gs1", "dataset_labels": []}],
+        "datasets": [{
+            "label": "alg1",
+            "data_dir": "fake",
+            "edge_files": [],
+            "other_files": [],
+            "node_files": []
+        }, {
+            "label": "alg2",
+            "data_dir": "faux",
+            "edge_files": [],
+            "other_files": [],
+            "node_files": []
+        }],
+        "gold_standards": [{
+            "label": "gs1",
+            "dataset_labels": [],
```

Comment:
Does this test config work when the gold standard is not matched with any dataset label?

Reply:
We can add this test case :+1:

Reply:
([Oh, there already was one](https://github.com/Reed-CompBio/spras/blob/ea59e4cec7d5197fd394f3704a5406facaa6f1a9/test/test_config.py#L234-L235))

Reply:
Good catch